### ISSUE TITLE
Add the minimize to Tray setting.

### ISSUE THREAD

**Issue by paolosezart** [monero-project/monero-gui#4205]
When the wallet is launched and syncs with the network, it takes up space on the taskbar. Is it possible to add a setting so that the program is minimized to the tray?

**Comment by selsta** [comment]
Do you mean wallet sync or daemon sync? You can close the wallet while the daemon syncs with the network.

**Comment by paolosezart** [comment]
I mean, so that the main window of the monero-gui program can be minimized to the tray, next to the clock. Now when collapsing it looks like this:
![GUI](https://github.com/monero-project/monero-gui/assets/38177890/6498458a-277f-4dd1-ba38-cdbea5c956a5)

I understand that you can just close the window, but then to start, you need to look for the program shortcut, enter the password again, and when you collapse into the tray, an instant launch is provided. Thank you.

**Comment by ThisNekoGuy** [comment]
I would also like this. Closing the flatpak release of the wallet just terminates everything rather than tucking it away out of sight.

**Comment by CaptLucy** [comment]
Yes I would also like this. Mostly I would like to be able to see the daemon is still running and be able to stop it via an icon in tray instead of having to use task manager or force dismounting the encrypted drive I keep it in, but it would also be nice if double clicking the icon opens the gui.

**Comment by tyrell-wallace** [comment]
Yes, please make this a feature.

**Comment by LeDechaine** [comment]
+1, yes this needs to be a feature.

Fun fact, I was searching for the same option, but I was using MATE, so for now I just ended up... doing that. Fullscreen in Workspace number 2... adding its icon to the "bottom-right tray". Temporary solution.
<img width="220" height="74" alt="Image" src="https://github.com/user-attachments/assets/12e2d175-2597-4797-b79b-4228101e9de9" />

This still needs to be a feature, for other desktop environments too.

**Comment by AllergicMushroom** [comment]
I'd also appreciate this.
